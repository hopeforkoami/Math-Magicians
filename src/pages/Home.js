import React from 'react';
import Header from '../components/Header';

const Home = () => (
  <div className="container">
    <Header selected_menu="home" />
    <section className="section home_section">
      <h1>Welcome to our page!</h1>
      <div className="section_content">
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
          tempor incididunt ut
          labore et dolore magna aliqua. Ut enim ad minim veniam, qui
          s nostrud exercitation ullamco
          laboris nisi ut aliquip ex ea commodo consequat.Duis aute i
          rure dolor in reprehenderit in
          voluptate velit esse cillum dolore eu fugiat nulla pariatur.
          Excepteur sint occaecat
          cupidatat non proident, sunt in culpa qui officia deserunt
          mollit anim id est laborum.
        </p>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed d
          o eiusmod tempor incididunt ut
          labore et dolore magna aliqua. Ut enim ad minim veniam, quis nos
          trud exercitation ullamco
          laboris nisi ut aliquip ex ea commodo consequat.Duis aute irur
          e dolor in reprehenderit in
          voluptate velit esse cillum dolore eu fugiat nulla pariatu
          r. Excepteur sint occaecat
          cupidatat non proident, sunt in culpa qui officia deserunt molli
          t anim id est laborum.
        </p>
      </div>

    </section>
  </div>
);

export default Home;
