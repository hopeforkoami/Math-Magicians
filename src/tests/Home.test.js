import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../pages/Home';

test('render Home component', () => {
  const { asFragment } = render(<Router><Home/></Router>);
  expect(asFragment()).toMatchSnapshot();
});
