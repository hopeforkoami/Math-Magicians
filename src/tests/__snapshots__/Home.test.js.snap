// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Home component 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <header>
      <div
        class="project_title"
      >
        Math Magicians
      </div>
      <nav>
        <ul
          class="menu"
        >
          <a
            href="/"
          >
            <li
              class="menu_element start_menu_element selected_menu_element"
            >
              Home
            </li>
          </a>
          <a
            href="/calcul"
          >
            <li
              class="menu_element"
            >
              Calculator
            </li>
          </a>
          <a
            href="/quote"
          >
            <li
              class="menu_element"
            >
              Quote
            </li>
          </a>
        </ul>
      </nav>
    </header>
    <section
      class="section home_section"
    >
      <h1>
        Welcome to our page!
      </h1>
      <div
        class="section_content"
      >
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, qui s nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.Duis aute i rure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed d o eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nos trud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.Duis aute irur e dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatu r. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt molli t anim id est laborum.
        </p>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
